docs/src/operations/requirements.md

```diff
@@ -18,12 +18,19 @@ transaction for each block the validator agrees with, which can cost up to
 
 The hardware recommendations below are provided as a guide.  Operators are encouraged to do their own performance testing.
 
+**NOTE**: These recommendations may be out of date as the network grows and resource requirements change.
+
 - CPU
   - 12 cores / 24 threads, or more
   - 2.8GHz base clock speed, or faster
   - SHA extensions instruction support
     - AMD Gen 3 or newer
+      - Known to work:
+        - **Bare minimum**: Epyc Gen 2: 7402
+        - **Current baseline**: Epyc Gen 3: 7443p, 7313, 74F3
+        - **Recommended for Futureproofing**: Epyc Gen 4: 9554, 9354, 9274, 9174. TR: 7965wx
```

Comment:
Rather than give specfic models, I think it would b e good to add a note that tells operators to prefer a higher clock speed over more cores.

Reply:
Updated in abfdac7076